tests/data/test_model.py

```diff
@@ -0,0 +1,83 @@
+"""Test for ModelData class"""
+
+from pathlib import Path
+import shutil
+
+from aiida_mlip.data.model import ModelData
+
+
+def test_local_file():
+    """Testing that the local file function works"""
+    # Construct a ModelData instance with the local file
+    model_path = Path("./tests/input_files/model_local_file.txt")
```

Comment:
May not want to start from `.` if run from elsewhere. `Path(__file__).parent` references the directory which contains this file